data_sources.py

```diff
@@ -0,0 +1,49 @@
+from functools import lru_cache
+import pandas as pd
+import numpy as np
+
+btax_data_path = 'btax_data/'
+
+
+@lru_cache(maxsize=None)
```

Comment:
@lucassz this cache wrapper looks neat. What's it doing and what's the motivation for using it here?